### ISSUE TITLE
Node School future events

### ISSUE THREAD

**Issue by avin45h** [nodeschool/Delhi-NCR#12]
@kushal-likhi and @suroorwijdan 
Let's plan an event at springboard in around August.

**Comment by kushal-likhi** [comment]
Yeah,, i can sponsor the event at springboard with food + drinks. :)

**Comment by suroorwijdan** [comment]
@kushal-likhi +1 from my side too.

**Comment by sahilchitkara** [comment]
Even i am in line for that in UrbanClap

**Comment by manoj-nama** [comment]
Guys, I see a lot of traction for the `4th July` event, we have filled the slots and cannot adjust any more people, yet there are lot of queries still. So maybe a bigger event next time would be great. 
Also, a ballpark time when we can have the next event would be awesome. Thoughts?

**Comment by avinash-jabong** [comment]
:+1: Springboard! Let's have the big one there

**Comment by kushal-likhi** [comment]
Awesome.... Lets do `NodeConf - Garage` next time .. ;)

**Comment by avinash-jabong** [comment]
:100:  Let's get this one right :) @kushal-likhi  nodeconf in next 6 months maybe? that is a big thing :dancer:

**Comment by kushal-likhi** [comment]
Nodeschool another event ?? August last saturday?

**Comment by manoj-nama** [comment]
August last Saturday is `rakhi`, i don't think people will be willing to leave their sisters and brothers that day .. ;)

**Comment by harishgadiya** [comment]
I thing meetup will be ok in afternoon

On Sunday, August 9, 2015, Manoj Nama <notifications@github.com> wrote:

> August last Saturday is rakhi, i don't think people will be willing to
> leave their sisters and brothers that day .. ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nodeschool/Delhi-NCR/issues/12#issuecomment-129192755>
> .
>


-- 
Sent from my iPhone

**Comment by shubhambatra** [comment]
you can set on Independence Day or next Day

**Comment by avinash-jabong** [comment]
Let's target somewhere in september and have a better audience? Let's plan it well, it became more of like a coaching last time. Plan for somewhere in September, Start planning today?

**Comment by tarungarg546** [comment]
First Saturday of September...5th September...
> Teacher's Day
Would be awesome and count me in too...and tell me more how to join this group and all.

**Comment by AbhimanyuAryan** [comment]
I am a transitioning full stack developer from RubyOnRails to Node.js 

NodeJs(on Twitter helped to come here :+1: ) 

btw don't see the Delhi group on Gitter. nodeschool/hyderabad is there. Please let know If there's a group......

And Yes plan for September. @tarungarg546 not 5 september. 13th September is cool. Its my birthday. We will have pizza party along with coding :+1:

**Comment by tarungarg546** [comment]
Its too late now and seems that this group is dead now...

**Comment by manoj-nama** [comment]
@avin45h @kushal-likhi @suroorwijdan @sahilchitkara  You guys wanna organize one soon at Springboard? It's been a quite long pause.

**Comment by AbhimanyuAryan** [comment]
@tarungarg546 no worries I am here. Let other join :+1:

**Comment by avinash-jabong** [comment]
@tarungarg546  @manoj-nama @AbhimanyuAryan @kushal-likhi 
The group ain't dead, Just been dealing with some major health issues, plus a few of us are busy with our fresh businesses. I hope it is understandable.
And yep, we can organize one at springboard. Im in.....

**Comment by tarungarg546** [comment]
hey @avinash-jabong its perfectly understandable. Health is everything.
Now i have shifted to bangalore and cant join your next meet.

**Comment by avinash-jabong** [comment]
@tarungarg546 You can catch up with bangalore guys too then. Nodeschool is everywhere :)

**Comment by manoj-nama** [comment]
Hey guys,

Since we have no upcoming slots for any future event (_looks like springboard is not gonna happen_). So we are thinking about an event in `1st` or `2nd` week of may.

Any thoughts on that? We need to move on it quickly.

**Comment by avinash-jabong** [comment]
@manoj-nama  sure, TTN?

**Comment by manoj-nama** [comment]
Yes, Don't have any other spaces i'm afraid.

**Comment by ravisuhag** [comment]
I can offer space. Our office can accommodate around 25-30 people. It is close to Sikandarpur metro station gurgaon. Let me know. Thanks

**Comment by avinash-jabong** [comment]
@manoj-nama TTN is actually a good option :) @ravisuhag Hey, thanks for the option.
So, as of now we have two options
- Nama's TTN
- Ravi's Office
I'll be talking it out at my office too, let's hope it works out. Let's close it today EOD. @manoj-nama What was the attendance of the last session?

**Comment by avinash-jabong** [comment]
Guys, @kushal-likhi is ready to join too, just had a word with him. Let's have an online discussion at some common time today :) The strength of volunteers is looking real good. What time are you guys comfortable with?

**Comment by manoj-nama** [comment]
@avin45h, @kushal-likhi @suroorwijdan We are doing the next event at TTN only, and have decided the date as 4th June (*Tentative*).

**Comment by avinash-jabong** [comment]
Thanks for the headsup, I'll fix the push issue as I am home.

P.S. : It would be very much appreciated if we plan things together next time (as a community).
